roles/davinci_resolve/DaVinciResolve.md

```diff
@@ -24,6 +24,12 @@ The DaVinci Resolve installation is disabled by default. To enable it:
    ```bash
    make install TAGS=davinci_resolve
    ```
+   or If the davinci resolve zip file already installed, add extra_vars to the make file
```

Comment:
Hi @Mohido, thank you for your PR!

I have some comments about your changes. The first one is about referring the zip archive as "already installed", I would phrase it differently, something like:

> or, if you have already downloaded the Davinci Resolve ZIP archive, you can avoid re-downloading it by pointing the script to the existing file like this: